patches/net/minecraft/world/item/alchemy/Potion.java.patch

```diff
@@ -0,0 +1,39 @@
+--- a/net/minecraft/world/item/alchemy/Potion.java
++++ b/net/minecraft/world/item/alchemy/Potion.java
+@@ -24,6 +_,8 @@
+     private final String name;
+     private final List<MobEffectInstance> effects;
+     private FeatureFlagSet requiredFeatures = FeatureFlags.VANILLA_SET;
++    private it.unimi.dsi.fastutil.objects.ReferenceSet<net.neoforged.neoforge.flag.Flag> requiredFlags = new it.unimi.dsi.fastutil.objects.ReferenceOpenHashSet<>();
++    private java.util.Set<net.neoforged.neoforge.flag.Flag> requiredFlagsView = it.unimi.dsi.fastutil.objects.ReferenceSets.unmodifiable(requiredFlags);
```

Comment:
Like with the mob effects, is there a measurable benefit to reusing this versus just creating it on-demand?